sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/AudioTranscriptionOptions.java

```diff
@@ -19,15 +17,13 @@ public final class AudioTranscriptionOptions {
      * The audio data to transcribe. This must be the binary content of a file in one of the supported media formats:
      * flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, webm.
      */
-    @JsonProperty(value = "file")
```

Comment:
Jackson annotation is removed by codegen, as this class is for multipart/form-date serialization, not JSON serializaiton.